src/main/java/com/mineboundteam/minebound/crafting/ArmorForgeRecipe.java

```diff
@@ -0,0 +1,72 @@
+package com.mineboundteam.minebound.crafting;
+
+import com.mineboundteam.minebound.registry.RecipeRegistry;
+import net.minecraft.core.NonNullList;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.Container;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.crafting.Ingredient;
+import net.minecraft.world.item.crafting.Recipe;
+import net.minecraft.world.item.crafting.RecipeSerializer;
+import net.minecraft.world.item.crafting.RecipeType;
+import net.minecraft.world.level.Level;
+import org.jetbrains.annotations.NotNull;
+
+public class ArmorForgeRecipe implements Recipe<Container> {
+    private final NonNullList<Ingredient> ingredients;
+    private final ResourceLocation resourceLocation;
+    private final ItemStack output;
+
+    public ArmorForgeRecipe(ResourceLocation resourceLocation, NonNullList<Ingredient> ingredients, ItemStack output) {
+        this.resourceLocation = resourceLocation;
+        this.output = output;
+        this.ingredients = ingredients;
+    }
+
+    @Override
+    public @NotNull ItemStack assemble(@NotNull Container pContainer) {
+        return output;
+    }
+
+    @Override
+    public boolean canCraftInDimensions(int pWidth, int pHeight) {
+        return true;
+    }
+
+    @Override
+    public @NotNull ResourceLocation getId() {
+        return resourceLocation;
+    }
+
+    @Override
+    public @NotNull NonNullList<Ingredient> getIngredients() {
+        return ingredients;
+    }
+
+    @Override
+    public @NotNull ItemStack getResultItem() {
+        return output.copy();
+    }
+
+    @Override
+    public boolean matches(@NotNull Container container, Level level) {
+        if (level.isClientSide) return false;
```

Comment:
will this mess up JEI/recipe book (if/when we end up implementing it)

Reply:
Nope. The tutorial I copied this from had implemented JEI integration beforehand.